docs/developer/client.md

```diff
@@ -207,27 +178,26 @@ persistent applications.
     typically `SIGINT` delivered by the PaaS or `Ctrl-C`, it
     de-registers with the API and gracefully shuts down.
 
-## Client state
+## SDM state
 
-A client, once registered, will continue to receive all the events it
+An SDM, once registered, will continue to receive all the events it
 has subscribed to until shuts down or one of the following scenarios
 occurs.
 
-### Multiple identical clients register
+### Multiple identical SDMs register
 
-If another client with the same name and version, typically obtained
-from the `package.json` "name" and "version" properties, registers,
-then all of the registered identical clients will receive the events
+If another client with the same name and version (typically obtained
+from the `package.json` "name" and "version" properties) registers,
```

Comment:
Personally, I think parentheses should be avoided. Good for LISP, bad for prose.